lib/plugins/body_reader.js

```diff
@@ -75,6 +75,9 @@ function bodyReader(options) {
                 return;
             }
 
+            if (bodyWriter.name === 'ub') {
```

Comment:
I want to minimize the changes and, align the text chunk handling with the other hand binary body's. I choose use bodywriter's name to distinct these two mode buffers.

At here, with check of bodywriter's name, I can know the buffer type. If buffer is actual utf8 mode text, then convert it to 'String' to compatible with intent behavior.